napari/layers/shapes/_shapes_utils.py

```diff
@@ -533,7 +534,23 @@ def triangulate_face(data):
         Px3 array of the indices of the vertices that will form the
         triangles of the triangulation
     """
-    vertices, triangles = PolygonData(vertices=data).triangulate()
+
+    try:
```

Comment:
Is it possible to concisely summarize the difference between this and [vispy's `Triangulation.triangulate`](https://cs.github.com/vispy/vispy/blob/26256fdc2574259dd227022fbce0767cae4e244b/vispy/geometry/triangulation.py?q=import%20triangle#L822)? And why we don't use that instead?

At the least, I think it would be useful to have some link to the vispy code and an explanation of what is different here.